### PR TITLE
Format homepage html

### PR DESCRIPTION
**Pull request by BobbyMcWho**
Solves https://github.com/mhwkb/mhwkb.github.io/issues/14

### CODE REVIEW

File: jekyll/_layouts/home.html [hunk 1]
```diff
@@ -1,92 +1,84 @@
----
-layout: default
----
-{% include category_display.html %}
+--- layout: default --- {% include category_display.html %}
```
Comment (andy5995) [inline jekyll/_layouts/home.html]: Looks like  your editor automatically made 4 lines into one. The page isn't rendering correctly when I test

File: jekyll/_layouts/home.html [hunk 2]
```diff
@@ -1,92 +1,84 @@
...
- basis; they are not UN staff and do not  receive a salary for their
-work. They are independent from any government or  organization and
-serve in their individual capacity. </p><p>For further information and media requests, please contact Ms. Dolores  Infante-Cañibano (+41 22 917 9768 / <a href="mailto:dinfante@ohchr.org">dinfante@ohchr.org</a>) </p><p>For <strong>media inquiries</strong> related  to other UN independent experts: <br>
-  Xabier Celaya, OHCHR Media Unit (+ 41 22  917 9383 / <a href="mailto:xcelaya@ohchr.org">xcelaya@ohchr.org</a>) </p><p>Concerned about the world we live in?&nbsp;Then<strong> STAND UP for someone’s rights today. &nbsp;</strong>#Standup4humanrights  and visit the web page at <a href="http://www.standup4humanrights.org/">http://www.standup4humanrights.org</a></p></div></span>
-    </div>
+    <span id="ctl00_PlaceHolderMain_DisplayNewsID_lblNewsFullText" class="lblnewsfulltext" style="display:inline-block;width:100%;">
+      <div class="ExternalClass4C9C8280F0EA4A2C9B9780FDF91D4465">
+        <p>GENEVA (6 June 2017) – The United Nations Special Rapporteur on the right to health, Dainius Pūras, has called for
```
Comment (andy5995) [inline jekyll/_layouts/home.html]: Hey, this looks a lot nicer! I should have been more specific in the ticket, and said 4 spaces for the indents. If you can fix it easily go ahead, if not, I'll take care of it later. @erichoog , we're going with 4 right?

**Comment by BobbyMcWho** [comment]
@andy5995 those should be resolved

**Comment by andy5995** [comment]
Thanks @BobbyMcWho

**Pull request merged**